procrastinate/tasks.py

```
@@ -167,6 +167,9 @@ def configure(
         ValueError
             If you try to define both schedule_at and schedule_in
         """
+        if self.app is None:
+            raise AssertionError("Tried to configure task whilst self.app was None")
```

(let's raise a custom exception here)

---

Created `UnboundTaskError`